docs/source/api/v2/deliveryservice_requests_id_status.rst

```diff
@@ -0,0 +1,177 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservice_requests-id-status:
+
+******************************************
+``deliveryservice_requests/{{ID}}/assign``
+******************************************
+Sets the status of a :term:`Delivery Service Request`.
+
+``GET``
```

Comment:
should this be a PUT

Reply:
Updated section heading to PUT in aa00b978b0.